org.virt_manger.virt-viewer.json

```diff
@@ -0,0 +1,321 @@
+{
+    "app-id" : "org.virt_manager.virt-viewer",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "3.28",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "virt-viewer",
+    "rename-desktop-file": "remote-viewer.desktop",
+    "rename-icon": "virt-viewer",
+    "finish-args" : [
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=wayland",
+        "--socket=pulseaudio",
+        "--share=network",
+        "--talk-name=org.a11y.*",
+        "--device=all",
+        "--filesystem=xdg-pictures",
+        "--filesystem=~/.ssh",
```

Comment:
There is also `--socket=ssh-auth` now, but I'm not sure if it's enough for ssh to work.

Reply:
I plan on testing this tonight.

Reply:
So, while the ssh-auth socket is enough for ssh to work on a basic level, if `~/.ssh` isn't passed through, it seemingly doesn't create a `known_hosts` file, and will always prompt the user whether they trust the fingerprint or not.